integration-tests/cypress/e2e/project/diagrams/general-view/directEditTests.cy.ts

```diff
@@ -94,6 +96,52 @@ describe('Node Creation Tests', () => {
         diagram.getNodes(diagramLabel, 'p1 :> aNewPart').should('exist');
         explorer.getExplorerView().contains('aNewPart');
       });
+
+      it.only('We can add properties to a new compartment item (that will be created) by direct editing the existing compartment item', () => {
```

Comment:
remove only (it looks like the CI doesn't detect it)

Reply:
Done